miner/worker.go

```diff
@@ -84,6 +84,30 @@ const (
 	staleThreshold = 7
 )
 
+// proofCache is cached in environment and holds data required for trace's storageProof and deletionProof
+type proofCache struct {
+	accountProof []hexutil.Bytes
+	storageProof map[string][]hexutil.Bytes
+	storageTrie  state.Trie
+	trieTracer   state.ZktrieProofTracer
```

Comment:
I notice that in `eth/tracers/api_blocktrace.go` we merge zktrieTracer for a block: 
```
env.zkTrieTracer[addrStr].Merge(zktrieTracer)
```

But we no longer do it here, is it because https://github.com/scroll-tech/go-ethereum/pull/321#discussion_r1199577951?

Reply:
the blocktrace would handle mutiple tracing parallelly so we need a scatter/gather pattern. i.e. assign a local tracer for in thread and merge the results to one. This is not need in the single thread miner worker: we can always use the same tracer in each tx.